services/QuillLMS/Gemfile.lock

```diff
@@ -238,16 +240,16 @@ GEM
     google-cloud-automl-v1 (0.3.0)
       gapic-common (~> 0.3)
       google-cloud-errors (~> 1.0)
-    google-cloud-automl-v1beta1 (0.4.0)
+    google-cloud-automl-v1beta1 (0.5.0)
```

Comment:
This gem, dependent on google-cloud-automl-v1, was updated by a  minor version ever after pinning. Should be ok.